kubermatic docs: Documenting for Seed Configuration (#1373)
kubermatic/docs: content/kubermatic/main/tutorials-howtos/administration/admin-panel/seed-configurations/_index.en.md

+++
title = "Seed Configurations"
date = 2023-02-20
weight = 20
+++

The Seed Configurations page in the Admin Panel allows administrators to see Seeds available in the KKP instance. This page presents a Seed's utilization broken down into Providers and Datacenters.

{{% notice note %}}
The Seed Configuration section is a readonly view page. This page does not provide any **CRUD** operations.
{{% /notice %}}

> - ### Seed Configurations

Reviewer: ```suggestion
### Seed Configurations
```

![Seed Configurations](/img/kubermatic/main/tutorials/seed-configurations/seed_confgurations.png?classes=shadow,border "Seed Configurations List View")

> - ### Seed Details

Reviewer: ```suggestion
### Seed Details
```

+++
title = "Seed Configurations"
date = 2023-02-20
weight = 20
+++

> Seed Configurations view in the Admin Panel allows administrators to see available seeds
> in KKP. This view provides listing of seeds, providers, datacenters and clusters associated to datacenter. All of these will be described below.

Reviewer: ```suggestion
The Seed Configurations view in the Admin Panel allows administrators to see Seeds available in KKP.
The view presents data about utilisation of Seeds, Providers and Datacenters. The details are described below.
```

> Seed Configurations view in the Admin Panel allows administrators to see available seeds
> in KKP. This view provides listing of seeds, providers, datacenters and clusters associated to datacenter. All of these will be described below.

Reviewer: ```suggestion
The Seed Configurations page in the Admin Panel allows administrators to see Seeds available in the KKP instance. This page presents a Seed's utilization broken down into Providers and Datacenters.
```

{{% notice note %}}

> Seed Configuration section is readonly view page. This page does not provider any **CRUD** operation.

Reviewer: ```suggestion
The Seed Configuration section is a readonly view page. This page does not provide any **CRUD** operations.
```

{{% /notice %}}

> - ### Listing seeds in KKP

Reviewer: ```suggestion
- ### Listing Seeds in KKP
```

![Seed Configurations](/img/kubermatic/main/tutorials/seed-configurations/seed_confgurations.png?classes=shadow,border "Seed Configurations List View")

> Following view displays Summary of how many following are linked

Reviewer: ```suggestion
Following view displays a summary of how many of following are linked
```

- Providers
- Datacenters
- Clusters

> - ### Listing of Providers

Reviewer: ```suggestion
- ### Providers
```

> Following view displays details of seed and following table displays list of providers and displays summary of associated in 
> 
> 
> - Datacenters
> - Clusters

Reviewer: ```suggestion
The following page presents Seed's statistics along with tables of utilization broken down respectively per Providers and Datacenters.
```

![Providers](/img/kubermatic/main/tutorials/seed-configurations/seed_confgurations_details.png?classes=shadow,border "Available providers per seed")

> In order to view breakdown for datacenters click on to expansion panel for each provider.
> 
> - ### Listing of Datacenters per provider
> 
> Expansion panel shows list of each datacenter name and associated cluster per datacenter.

Reviewer: We don't need to be that specific. 
```suggestion
```
